File 5.41 changed the MIME type of Python bytecode; adjust accordingly.

Taken from upstream:

  https://github.com/ahupp/python-magic/commit/0ae7e7ceac0e80e03adc75c858bb378c0427331a

diff --git a/test/test.py b/test/test.py
index 0c4621c..e443b84 100755
--- a/test/test.py
+++ b/test/test.py
@@ -90,7 +90,7 @@ def test_mime_types(self):
         try:
             m = magic.Magic(mime=True)
             self.assert_values(m, {
-                'magic._pyc_': ('application/octet-stream', 'text/x-bytecode.python'),
+                'magic._pyc_': ('application/octet-stream', 'text/x-bytecode.python', 'application/x-bytecode.python'),
                 'test.pdf': 'application/pdf',
                 'test.gz': ('application/gzip', 'application/x-gzip'),
                 'test.snappy.parquet': 'application/octet-stream',
